src/msal-client-credentials-flow/README.md

```diff
@@ -0,0 +1,170 @@
+  "isFallbackPublicClient": true,
+  "notes": null,
+  "publisherDomain": "contoso.onmicrosoft.com",
+  "serviceManagementReference": null,
+  "signInAudience": "AzureADMyOrg",
+  "tags": [],
+  "tokenEncryptionKeyId": null,
+  "defaultRedirectUri": null,
+  "certification": null,
+  "optionalClaims": null
+  ...
+}
+```
+## Prerequisites
+
+- An Azure Active Directory (Azure AD) tenant. You can [open an Azure account for free](https://azure.microsoft.com/free) to get an Azure AD instance.
+- [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+
+## Setup
+
+### 1. Register the app
+
+First, complete the steps in [Quickstart: Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the web API.
+
+Use these settings in your app registration.
+
+| App registration <br/> setting    | Value for this sample app                                                    | Notes                                                                                              |
+|---------------------------------:|:------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| **Name**                          | `active-directory-dotnet-minimal-api-aspnetcore-client-credentail-flow`      | Suggested value for this sample. <br/> You can change the app name at any time.                    |
+| **Supported account types**       | **Accounts in this organizational directory only (Single tenant)**           | Suggested value for this sample.                                                                   |
+| **Platform type**                 | _None_                                                                       | No redirect URI required; don't select a platform.                                                                    |
+| **Client secret**                 | _**Value** of the client secret (not its ID)_                                | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).        |
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Configure the web API
+
+1. Open the _~/msal-client-credentials-flow/appsettings.json_ file in your code editor and modify the following values values with those from your [app's registration in the Azure portal](https://docs.microsoft.com/azure/active-directory/develop/quickstart-configure-app-expose-web-apis):
+
+   ```json
+   "ClientId": "Enter_the_Application_Id_here",
+   "TenantId": "Enter_the_Tenant_Info_Here",
+   "ClientSecret": "Enter_the_Application_CLient_Secret_Here",
+   ...
+   "RelativePath": "Enter_the_Application_Object_Id_Here",
+   ...
```

Comment:
we don't have one at the front of this section, probably don't need one at the rear.

```suggestion
```

Reply:
I got you @ckittel 

done | addressed from [ce25b](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/22/commits/ce25bf07305d024c8c37477c55b74b0771b293f9)

Reply:
Ah, you removed them both.  I suppose that's fine.  I liked the middle one so the user knew to "look down further" -- but do you think it'll be okay without that?

Reply:
you got it @ckittel 

done | address from [76ecb](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/22/commits/76ecb812d0404227278f08c62fc3144fe9885093)